app/src/main/java/io/gnosis/safe/ui/transactions/details/view/MultisendActionView.kt

```diff
@@ -0,0 +1,27 @@
+package io.gnosis.safe.ui.transactions.details.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.constraintlayout.widget.ConstraintLayout
+import io.gnosis.safe.databinding.ViewMultisendActionBinding
+import io.gnosis.safe.utils.abbreviateEthAddress
+import pm.gnosis.crypto.utils.asEthereumAddressChecksumString
+import pm.gnosis.model.Solidity
+
+class MultisendActionView @JvmOverloads constructor(
```

Comment:
Should we not be using `LabeledAddressItem`? I think we are creating too many custom views, maybe we should start trying to generalise a bit more?

Reply:
This view has different layout and shows not only blockie, address, but also method and potentially (in v3) tx type icon. I think we have custom views that are specific enough for their task and I would not generalize them at this point in time